src/Skoruba.IdentityServer4.Admin/Startup.cs

```diff
@@ -11,40 +11,28 @@
 using Skoruba.IdentityServer4.Admin.EntityFramework.Shared.DbContexts;
 using Skoruba.IdentityServer4.Admin.EntityFramework.Shared.Entities.Identity;
 using Skoruba.IdentityServer4.Admin.Helpers;
+using Skoruba.IdentityServer4.Admin.Configuration;
+using Skoruba.IdentityServer4.Admin.Configuration.Constants;
 
 namespace Skoruba.IdentityServer4.Admin
 {
     public class Startup
     {
-        public Startup(IWebHostEnvironment env)
+        public Startup(IWebHostEnvironment env, IConfiguration configuration)
         {
             JwtSecurityTokenHandler.DefaultInboundClaimTypeMap.Clear();
-
-            var builder = new ConfigurationBuilder()
```

Comment:
Because you don't need to create an instance of IConfiguration by yourself.
It was already done by `Host.CreateDefaultBuilder(string args[])` ([src](https://github.com/aspnet/Extensions/blob/7a85adc13f3d2fe7712a7bfd3cef8691ea78ba61/src/Hosting/Hosting/src/Host.cs#L70-L92)).
So you can just get that instance from DI container